tests/installation/user_settings.pm

```diff
@@ -15,6 +15,19 @@ use strict;
 use warnings;
 use parent qw(installation_user_settings y2logsstep);
 use testapi;
+use utils qw(type_string_slow);
+
+
+sub enter_userinfo {
+    my ($self, %args) = @_;
+    send_key 'alt-f';    # Select full name text field
+    wait_screen_change { $args{retry} ? type_string_slow $realname : type_string $realname };
+    send_key 'tab';      # Select password field
+    send_key 'tab';
+    $self->type_password_and_verification;
+    # ensure that the screen is not changing
+    wait_still_screen 3, 15;
```

Comment:
hm, that means we wait longer in all tests which do not show the previous problem and also the time might not be enough e.g. on slow environments where there is no mistyping but it just takes longer to finish typing? I would prefer a solution with a proper `assert_screen` and the usual timeout.

Reply:
not necessarily, before we just assumed that the screen is not changing anymore and the needle is matching within 30s.
so this already helps to make it more robust and less error-prone